static/src/javascripts-legacy/projects/common/modules/ui/selection-sharing.js

```diff
@@ -74,15 +74,16 @@ define([
 
         if (selection && selection.rangeCount > 0 && selection.toString()) {
             range = selection.getRangeAt(0);
-            rect = clientRects.getBoundingClientRect(range);
-            top = $body.scrollTop() + rect.top;
-            twitterMessage = range.toString();
 
             if (!isValidSelection(range)) {
                 hideSelection();
                 return;
             }
 
+            rect = clientRects.getBoundingClientRect(range);
```

Comment:
This function [can return `null`](https://github.com/guardian/frontend/blob/471f87c571656772ba5f884d92fc62c81b9efd41/static/src/javascripts-legacy/lib/client-rects.js#L75) and so it may be the origin of the problem too.